blockbuster/lanes/free/lane.go

```diff
@@ -26,7 +26,7 @@ func NewFreeLane(cfg blockbuster.BaseLaneConfig, factory Factory) *Lane {
 	}
 
 	return &Lane{
-		DefaultLane: base.NewDefaultLane(cfg),
+		DefaultLane: base.NewDefaultLane(cfg).WithName(LaneName),
```

Comment:
This was added mostly for logging reasons. before, the subclass of the default lane would by default inherit the name so we have to manually override it here.